specutils/analysis/flux.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import division
+
+import numpy as np
+from astropy.units.quantity import Quantity
+from .utils import computation_wrapper
+
+
+__all__ = ['line_flux', 'equivalent_width']
+
+
+def line_flux(spectrum, region=None):
+    """
+    Computes the line flux over a spectrum. Applies a region if given.
+
+    Parameters
+    ----------
+    spectrum : Spectrum1D
+        The spectrum object over which the line flux will be calculated.
+
```

Comment:
missing ``region`` keyword.  Also, shouldn't it be ``regions`` if it can be multiple regions?